src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
```

Comment:
```suggestion
      action="#"
```
change it and find out the difference
or you can put the link from the previous task